src/literal.h

```diff
@@ -135,10 +135,7 @@ class Literal {
     assert(type == Type::funcref);
     return func;
   }
-  const ExceptionPackage& getExceptionPackage() const {
-    assert(type == Type::exnref);
-    return *exn.get();
-  }
+  ExceptionPackage getExceptionPackage() const;
```

Comment:
why move this body to the cpp? all the siblings are here it seems.

Reply:
Since `ExceptionPackage` is now returned by value, it needs to be a complete type at the point of this function definition. But the definition of `ExceptionPackage` can't move before the definition of `Literal` for the same reason, so the only choice was to move `getExceptionPackage` to be defined out of line, which means it needs to be in the .cpp file.

Reply:
I see, thanks...